client-misk/src/main/kotlin/app/cash/backfila/client/misk/BackfilaModule.kt

```diff
@@ -44,5 +51,21 @@ class BackfilaModule(
       .build()
 }
 
+private fun mapBinder(binder: Binder) = MapBinder.newMapBinder(
+    binder,
+    object : TypeLiteral<String>() {},
+    object : TypeLiteral<KClass<out Backfill<*, *>>>() {},
+    ForBackfila::class.java
+)
+
+inline fun <reified T : Backfill<*, *>> Binder.bindBackfill() {
```

Comment:
is there a smarter way to do this?

Reply:
I'm no expert but I think that's right.

Reply:
Maybe @swankjesse has a suggestion but I believe this is correct.

Reply:
I prefer a slightly different syntax where we let the application itself do the multibind() call.

Here’s how we document it in StaticResourceEntry:
```
/**
 * This data class is used with Guice multibindings. Register instances by calling `multibind()`
 * in a `KAbstractModule`:
 *
 * ```
 * multibind<StaticResourceEntry>().toInstance(StaticResourceEntry(...))
 * ```
 */
class StaticResourceEntry(
  url_path_prefix: String = "/",
  private val resourcePath: String
) : ValidWebEntry(url_path_prefix = url_path_prefix) {
 ...
}
```

Reply:
Or perhaps what we do for Web Actions?
```
    install(WebActionModule.create<InternalErrorAction>())
```

Reply:
(this way you don’t need to subclass a special module class, which I find a bit awkward)

Reply:
thanks, that's way better